### PR TITLE
added CNNmodel: 1mmT1w [skip ci]

### PR DESCRIPTION
**Pull request by akhanf**

### CODE REVIEW

File: hippunfold/config/snakebids.yml [hunk 1]
```diff
@@ -64,6 +64,19 @@ pybids_inputs:
       - acquisition
       - run
 
+  1mmT1w:
```
Comment (akhanf) [inline hippunfold/config/snakebids.yml:67]: This part isn't needed, as that would mean we would be grabbing a new bids type called 1mmT1w.. We are just using the same T1w image in our case..

**Comment by akhanf** [comment]
@jordandekraker can you give this a try? It should still use the T1w image as usual, but when --modality 1mmT1w is set the cnn model will be different

**Comment by jordandekraker** [comment]
still getting a similar issue to when I was trying things out earlier: 
```
(venv) [jdekrake@gra799 hippdev]$ hippunfold /project/6050199/akhanf/ext-bids/hcp_aging/bids /scratch/jdekrake/hcp_aging_1mmtest/ participant --modality 1mmT1w --participant_label 6002236 --cores all
KeyError in line 21 of /scratch/jdekrake/hippdev/hippunfold/hippunfold/workflow/Snakefile:
'1mmT1w'
  File "/scratch/jdekrake/hippdev/hippunfold/hippunfold/workflow/Snakefile", line 21, in <module>
  File "/scratch/jdekrake/hippdev/venv/lib/python3.7/site-packages/snakebids/__init__.py", line 360, in generate_inputs_config
  File "/scratch/jdekrake/hippdev/venv/lib/python3.7/site-packages/snakebids/__init__.py", line 236, in get_input_config_from_bids

Traceback (most recent call last):
  File "/scratch/jdekrake/hippdev/venv/bin/hippunfold", line 33, in <module>
    sys.exit(load_entry_point('hippunfold', 'console_scripts', 'hippunfold')())
  File "/scratch/jdekrake/hippdev/hippunfold/hippunfold/run.py", line 14, in main
    app.run_snakemake()
  File "/scratch/jdekrake/hippdev/venv/lib/python3.7/site-packages/snakebids/app.py", line 209, in run_snakemake
    run(snakemake_cmd)
  File "/scratch/jdekrake/hippdev/venv/lib/python3.7/site-packages/snakebids/app.py", line 41, in run
    raise Exception("Non zero return code: %d"%process.returncode)
Exception: Non zero return code: 1
```

**Comment by akhanf** [comment]
I was missing a couple things, is working for me with a dry run now -- 
however I did just notice we will also have to pull the latest autotop; will do that in this branch in a sec too, so we can confirm nothing gets broken

**Comment by akhanf** [comment]
ok @jordandekraker can give it a try now to see how it works with `--modality 1mmT1w`

**Comment by jordandekraker** [comment]
Yup, its working on my end!
I'm not authorized to merge this PR, but when you get the chance and the docker container is rebuilt let me know so Konrad's lab can try this model.

**Pull request merged**